Solution1/BusinessLayer/Extensions/WeatherDtoExtensions.cs

```diff
@@ -11,13 +11,8 @@ public static string GetStringRepresentation(this WeatherDTO weatherDTO)
 
         public static WeatherDTO FillCommentByTemp(this WeatherDTO weatherDTO)
```

Comment:
make extension for all inherited from BaseWeatherDTO

Reply:
+